app/controllers/v0/claims_controller.rb

```diff
@@ -4,13 +4,18 @@ class ClaimsController < ApplicationController
     skip_before_action :authenticate
 
     def index
-      render json: current_user.claims,
+      render json: Claim.fetch_all(current_user.vaafi_headers),
```

Comment:
Would it make sense for this to be `render json: Claim.fetch_all(current_user),` and then we would give the claim model the responsibility of calling `user.vaafi_headers`? Ultimately, this is going to be pulled out of the session store, right?